### DISCUSSION TITLE
Using Notifications with Drawers

### DISCUSSION THREAD

**Discussion by noahsb** [mantinedev/mantine#3868] [Q&A] [answered]
Hello, thank you for all your hard work maintaining Mantine!

In my app, I use the `Drawer` component a lot to render forms on the side so users can see some of the details they're inputting if they forget. I use `showNotifications` to display success and error messages after the form has been submitted. This works fine with success messages because the `Drawer` closes upon success, but when failures occur and I display an error notification on the screen and the `Drawer` is still open, it is set to `aria-hidden` (because all content outside of the `Drawer` is set to `aria-hidden`).

Is there a way around this? I'd like our users with screen readers to be able to see notifications in a `Drawer`. This would also make testing easier by not adding a `hidden: true` every time

Summary: Content outside of `Drawers` are set to `aria-hidden` making notifications not visible to screen readers

**Comment by rtivital** [comment, marked as answer]
I do not think that it is currently possible, it is a part of focus trap hook that cannot be configured
